code/modules/vehicles/ridden/tug.dm

```diff
@@ -0,0 +1,47 @@
+/obj/vehicle/ridden/tug
```

Comment:
doesnt this file need adding to the .dme?
You dont need to add .dmi filee, but this one is a .dm file

Reply:
Was having merge conflicts earlier and took it off the .dme list temporarily, thank you for reminding me to put it back on XD